contracts/asset-proxy/contracts/src/interfaces/IWallet.sol

```diff
@@ -0,0 +1,38 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+pragma experimental ABIEncoderV2;
+
+
+contract IWallet {
+
+    bytes4 internal constant LEGACY_WALLET_MAGIC_VALUE = 0xb0671381;
```

Comment:
I guess we can't import this due to circular dependencies?

Reply:
Yep. The alternative is to have `exchange` import this file from the `asset-proxy` package, but that just *feels* wrong.

Reply:
We should probably create a task for moving this to `exchange-libs`, but doesn't have to be in this PR.